plugins/file/plugin.go

```diff
@@ -21,16 +26,76 @@ func init() {
 	plugins.RegisterPlugin("file", setupFile6, setupFile4)
 }
 
+//Record holds an IP lease record
+type Record struct {
+	IP        net.IP
+	leaseTime uint32
```

Comment:
This represents a duration, how about using a int64 or time.Duration. https://golang.org/pkg/time/#Duration